cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapStatusEvaluator.java

```diff
@@ -75,4 +94,17 @@ public CareGapsStatusCode getGapStatus(Measure measure, MeasureReport measureRep
 
         return CareGapsStatusCode.CLOSED_GAP;
     }
+
+    /*
+    // TODO implement Measure Group Level improvement notation extension
+    private boolean groupHasImprovementNotation(MeasureReportGroupComponent groupComponent) {
+        return groupComponent.getExtensionByUrl(MEASUREREPORT_IMPROVEMENT_NOTATION_EXTENSION) != null;
+    }
```

Comment:
Do you plan on leaving all this code commented out?

Reply:
temporarily, it will merge in within the next couple days